.github/workflows/ci.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: '16'
```

Comment:
I don’t think it’s a good idea to pin to a non-major version since those aren’t guaranteed to be LTS.

Reply:
18 then